x/did/types/msgs.go

```diff
@@ -0,0 +1,99 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
+	_ sdk.Msg = &MsgCreateDID{}
+	_ sdk.Msg = &MsgUpdateDID{}
+)
+
+// MsgCreateDID defines a CreateDID message.
+type MsgCreateDID struct {
+	DID          DID            `json:"did"`
+	Document     DIDDocument    `json:"document"`
+	OwnerAddress sdk.AccAddress `json:"owner_address"`
```

Comment:
Would something other than the owner be more appropriate?

Reply:
Makes sense. Changed it to `FromAddress`. 👍